components/script/dom/globalscope.rs

```diff
@@ -3255,6 +3261,25 @@ impl GlobalScope {
         let _ = self.console_group_stack.borrow_mut().pop();
     }
 
+    pub(crate) fn increment_console_count(&self, label: &DOMString) -> usize {
+        let mut map = self.console_count_map.borrow_mut();
+        if let Some(val) = map.get_mut(label) {
+            *val += 1;
+            return *val;
+        }
+        map.insert(label.clone(), 1);
+        1
```

Comment:
I think this could use the Entry API. Perhaps something like the following works?

```suggestion
        self.console_count_map.borrow_mut().entry(label)
             .and_modify(|e| { *e += 1 })
             .or_insert(1);
```